recipe/meta.yaml

```diff
@@ -1,29 +1,31 @@
+{% set name = "packaging" %}
 {% set version = "24.2" %}
+{% set python_min = "3.8" %}
 
 package:
-  name: packaging
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/packaging/packaging-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
It would be better to keep these as-is (minus the change to pypi.org, which is necessary). Putting the name behind a variable is useless indirection, and doubly so because it can never be changed.

Reply:
@h-vetinari I can revise these but I'm following what `grayskull` does at the moment

```console
$ pixi global install grayskull
$ grayskull pypi packaging
$ head -n 11 packaging/meta.yaml 
{% set name = "packaging" %}
{% set version = "24.2" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/packaging-{{ version }}.tar.gz
  sha256: c228a6dc5e932d346bc5739379109d49e8853dd8223571c7c5b55260edc0b97f

```

so maybe we should fix it there first.

Reply:
I know, and I disagree with that too (but not enough that I ever went and tried to get it fixed). 🤷‍♂️

Reply:
Fixed up in the rebase.